### DISCUSSION TITLE
What color manioultion tactic are you using for rgb->oklch

### DISCUSSION THREAD

**Discussion by tigawanna** [saadeghi/daisyui#3175] [Q&A] [answered]
Was trying to build a theme builder and i've been stuck trying to get oklch colors from the rgb in most color pickers and it's not working as expected , i also can't find any with oklch support
What method are you using when you load the js daisyui theme from the config into the css oklch variables?I aso saw you recommend another [theme builder](https://themes.ionevolve.com/) but couldn't find it's code as i was trying to store the changes in the url to make the themes more sharable

**Comment by saadeghi** [comment, marked as answer]
We're using this library for color conversion in both daisyUI package and daisyUI website:  
https://culorijs.org/

It supports RGB and HSL as well as modern formats like OKLCH
